src/components/academy/grading/index.tsx

```diff
@@ -25,6 +26,12 @@ type State = {
   columnDefs: ColDef[]
   filterValue: string
   groupFilterEnabled: boolean
+  missionFilter: boolean
```

Comment:
I feel its unnecessary to add so much clutter into the internal state for each filter. The whole reason behind using `ag-grd` instead of a pure table was to reduce this!